src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -611,10 +611,14 @@ class TcgStatics {
     bg().em().run(new CantEvolve(pcs, bg().getTurnCount()));
     bg().em().run(new ActivateAbilities((PokemonCard) card, pcs, reason));
   }
-  static devolve (PokemonCardSet pcs, Card card, CardList newLocation){
-    def blocked = bg().em().run(new MoveCard(card, newLocation));
-    if (blocked) {
-      return;
+  static devolve (PokemonCardSet pcs, Card card, CardList newLocation, Source source=ATTACK){
+    def blockedDevolve = bg().em().run(new Devolve(pcs, source))
+    if (blockedDevolve) {
+      return
+    }
```

Comment:
`Devolve` runs `CheckAbilities`. By the time it runs, the card has to be moved away, but this change prevents that, thus the abilities will be incorrect after this change.